src/adapters/fidelity.js

```diff
@@ -0,0 +1,103 @@
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+var STATUS = require('../constants').STATUS;
+
+var FidelityAdapter = function FidelityAdapter() {
+  var FIDELITY_BIDDER_NAME = 'fidelity';
+  var FIDELITY_SERVER_NAME = 'x.fidelity-media.com';
+
+  function _callBids(params) {
+    var bids = params.bids || [];
+    bids.forEach(function (currentBid) {
+      if (!document.MAX_used) document.MAX_used = ',';
```

Comment:
I'm curious about why a variable is being attached to the `document` object like this? Could an internal function variable be used instead?